src/Discovery/aggMDSUtils.jsx

```diff
@@ -75,7 +75,7 @@ const loadStudiesFromAggMDSRequests = async (offset, limit) => {
       x.short_name = x.name;
       x._unique_id = `${commonsName}_${x._unique_id}_${index}`;
       x.commons_url = commonsInfo.commons_url;
-      x.tags.push(Object({ category: 'Commons', name: commonsName }));
+      x.tags.push(Object({ category: 'Data Resources', name: commonsName }));
```

Comment:
change made by request of @nmetokishlubsky for HEAL, need to verify if we want to merge it back to master later

Reply:
This change has been reverted. HEAL envs needs to update portal config to assign a customized display name for this `Commons` category